.github/workflows/test.yml

```
@@ -0,0 +1,59 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+  pull_request:
+
+env:
+  RUBY_VERSION: 2.7.4
+  NODE_VERSION: 12.9.1
+    
+jobs:
+  test:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11
+        ports:
+          - 5432:5432
+        env:
+          RAILS_ENV: test
+          DATABASE_USERNAME: postgres
+          DATABASE_PASSWORD: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+          bundler-cache: true
+
+      - name: Setup & create Database
+        run: |
+          bundle exec rake decidim:generate_external_test_app
```

```suggestion
          bundle exec rake decidim:generate_external_test_app
```

no necessary anymore